codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/protocols/serialize/EventStreamMarshallerGenerator.kt

```diff
@@ -75,6 +76,37 @@ open class EventStreamMarshallerGenerator(
         }
     }
 
+    fun renderInitialMessageGenerator(contentType: String): RuntimeType {
+        return RuntimeType.forInlineFun("initial_message_from_body", eventStreamSerdeModule) {
+            rustBlockTemplate(
+                """
+                pub(crate) fn initial_message_from_body(
+                    body: #{SdkBody}
+                ) -> #{Message}
+                """,
+                *codegenScope,
+            ) {
+                rust("let mut headers = Vec::new();")
+                addStringHeader(":message-type", "\"event\".into()")
+                addStringHeader(":event-type", "\"initial-request\".into()")
+                addStringHeader(":content-type", "${contentType.dq()}.into()")
+                rustTemplate(
+                    """
+                    let body = bytes::Bytes::from(
```

Comment:
Not sure if we need to use `#{Bytes}` instead of using `bytes::Bytes`?

Reply:
Fixed in a727492.